packages/algob/package.json

```diff
@@ -99,6 +99,7 @@
     "source-map-support": "^0.5.19",
     "tmp": "^0.2.1",
     "yaml": "^1.10.0",
+    "mocha": "^9.0.3",
```

Comment:
I this should be devDependency

Reply:
Yes, but it is required in /src while executing `algob test` command. So putting this as a dev dependency would fail